ONSdigital/response-operations-ui
# RAS-1101: Bug in secure messaging pre prod - Message sends twice

# What and why?
When a message is sent and there is a delay, the submit button remains active. The submit can then be clicked 'X' amount of times leading to an equivalent 'X' amount of duplicates. The 'loader' variant has been added to the submit button. The submit button will no longer be active, a loading wheel will appear on the button and will prevent the button being clicked.

# How to test?

- Run acceptance tests
- Pull this PR
- Add 'time.sleep(30)' to https://github.com/ONSdigital/response-operations-ui/blob/3c540c5df10ea9241481515477207d86ed0f9229/response_operations_ui/controllers/message_controllers.py#L162 to cause a delay
- Run this PR locally
- Send a message in rOps
- The submit button should look like this 

<img width="163" alt="image" src="https://github.com/ONSdigital/response-operations-ui/assets/47788084/17e4780f-1710-43f9-b2ed-31d116460ed9">

-After the delay, you should be redirected to the inbox overview page

Additional testing:
-Deploy this PR to your env
-Run acceptance tests

# Jira
https://jira.ons.gov.uk/browse/RAS-1101
